examples/clob/programs/sp1/src/lib.rs

```diff
@@ -0,0 +1,16 @@
+use once_cell::sync::Lazy;
+use std::fs;
+
+pub const CLOB_ELF: &[u8] = include_bytes!("../program/elf/clob-sp1-guest");
+
```

Comment:
If we ditch the bincode, I think we could just do this, which would be awesome because we can then use this public const everywhere

```suggestion
pub const CLOB_ELF_ID: &[u8] = include_bytes!("../program/elf/clob-sp1-guest.vkey")
```

Reply:
If this makes sense, we should use the same pattern for all the programs